src/Haskell/Ide/Engine/Scheduler.hs

```diff
@@ -99,18 +99,18 @@ class HasScheduler a m where
 newScheduler
   :: IdePlugins
      -- ^ The list of plugins that will be used for responding to requests
-  -> GM.Options
+  -> BiosOptions
    -- ^ Options for the ghc-mod session. Since we only keep a single ghc-mod session
```

Comment:
`ghc-mod` -> `bios`?

Reply:
yes, we are opening the way for `hie-bios`. The idea is that the `BiosOptions` structure contains generally useful settings, that can then be interpreted appropriately by any backend.

So currently logging and any extra GHC command line params.

Reply:
No, I mean the comment should be updated, right?